src/System.Text.Encoding/tests/Encoder/EncoderConvert2.cs

```diff
@@ -38,7 +38,7 @@ public class EncoderConvert2
         #endregion
 
         #region Positive Test Cases
-        // PosTest1: Call Convert to convert a arbitrary character array with ASCII encoder
+        // PosTest1: Call Convert to convert a arbitrary character array with UTF8 encoder
```

Comment:
Nit: as long as you're touching all of these, consider giving the tests better names.  Not required, though; it's just debt we've yet to clean up.